docs-importer/README.md

```diff
@@ -25,7 +25,7 @@ For the next start, pass the `-d` argument to skip the wizard:
 
 ## Daemon mode
 
-The daemon mode scan the input directory every 30 seconds for new files. Once a file is found and imported, it is **deleted**.
+The daemon mode scan the input directory every 30 seconds for new files. Once a file is found and imported, it is **deleted**. You can set the "copyFolder" value in the preference file to copy the file before deletion.
```

Comment:
It would be nice if the daemon would also ask for it.

Reply:
I didn't add it since I thought it should not really be a default option.